tardis/simulation/base.py

```diff
@@ -383,6 +387,55 @@ def run(self):
         )
         self._call_back()
 
+        self.interactive_widget()
+
+    def interactive_widget(self):
+        """
+        Generates the Interactive Widget for displaying the 
+        Plasma Data that has been generated by the simulation for 
+        each iteration
+        """
+
+        # Creating the List for no of iterations
+        iteration_number = []
+        for iteration in range(self.iterations):
+            iteration_number.append(iteration)
+
+        # Instantiated the Dataframes for storing the Plasma Data
+        df = pd.DataFrame(index=np.arange(len(self.model.t_rad)))
+
+        # Populating the DataFrame for the Simulation Data for t_rad & w
+        for iteration in range(0, self.iterations):
+            df[f"t_rad[{iteration}]"] = self.iterations_t_rad[iteration]
+            df[f"w[{iteration}]"] = self.iterations_w[iteration]
+
+        # Instantiated the Dropdown Widget
+        drop_down = widgets.Dropdown(
+            options=iteration_number, value=0, description="Iteration No",
+        )
+
+        # Displays the t_rad & w values for the chosen iteration
+        def display_with_iteration(value):
+            df_output = df[[f"t_rad[{value}]", f"w[{value}]"]]
+            table_output = create_table_widget(
+                data=df_output,
+                col_widths=[6, 47, 47],
+                table_options={"maxVisibleRows": 10},
+            )
+            display(table_output)
+
+        # Instantiated the Output widget for displaying
+        out = widgets.Output()
+
+        # Checks for the change in the values of the Drop down list
+        def fun(ipw_obj):
```

Comment:
Not a descriptive function name

Reply:
Sure, changed it to something more tangible 😉